src/node_url.cc

```diff
@@ -1351,6 +1368,41 @@ namespace url {
                             v8::NewStringType::kNormal).ToLocalChecked());
   }
 
+  static void ToUSVString(const FunctionCallbackInfo<Value>& args) {
+    Environment* env = Environment::GetCurrent(args);
+    CHECK_GE(args.Length(), 2);
```

Comment:
All the existing functions use `CHECK_GE`, and I don't see a reason to be stricter than what this function actually uses.